USAGE.md

```diff
@@ -0,0 +1,36 @@
+# STARTTLS Policy List Usage Guidelines
+
+## MTA-STS interoperation for sending mailservers
```

Comment:
I guess that the `VALIDATION.md` file is meant to set forth what people need to know about "MTA-STS interoperation for receiving mailservers". However, since the names `USAGE` and `VALIDATION` don't directly make this clear to a new user, I suggest explicitly saying somewhere something like "For information about configuration of *receiving* mailservers and addition of a receiving mailserver to the list, please see the VALIDATION.md file". (Ideally, this would go nearby or right above here—maybe an Introduction section that says "This file describes how a *sending* mailserver can make use of the list to secure e-mail that it originates. For information about *receiving* mailservers and how to get them added to the list, please see VALIDATION.md instead".